src/modules/menu/components/menu/reportProblem/ImportFileLocal.vue

```diff
@@ -0,0 +1,109 @@
+<script setup>
+import { computed, ref, watch } from 'vue'
+import { useI18n } from 'vue-i18n'
+
+const LOCAL_UPLOAD_ACCEPT = '.kml,.gpx,.pdf,.zip,.jpg,.jpeg,.kmz'
+const LOCAL_UPLOAD_MAX_SIZE = 250 * 1024 * 1024 // 250mb
+
+const i18n = useI18n()
+
+const emits = defineEmits(['file-selected'])
```

Comment:
Same here as it seems to be the same code as the import kml file local, this should move to src/utils/component and be renamed as `InputLocalFile`, the `LOCAL_UPLOAD_ACCEPT` and `LOCAL_UPLOAD_MAX_SIZE` should be properties that the user can define, which would allow to reuse this component for other input file type. Also rename variables and attributes accordingly (e.g. ImportFileLocalInput => InputLocalFile, data-cy="import-file-local-content" => data-cy="input-file-local-content")